tests/deployment_validation/test_deployment.py

```diff
@@ -0,0 +1,12 @@
+from playwright.sync_api import Page, expect
+
+
+def test_move_task_to_column(validate_version: str, browser: Page) -> None:
```

Comment:
Why make a new test, and not use the ones we already have?

Reply:
It is a wrong name for the tests. I don't want to run a full E2E after deployment as we don't know what the state is. But I want to make sure the right version is deployed and run some basic tests, like make sure there are not javascript (startup) errors and whatever more we can test to assume 'deployment went well', all lights are green.